sourmash/command_sketch.py

```diff
@@ -0,0 +1,246 @@
+"""
+Functions implementing the 'sketch' subcommands and related functions.
+"""
+import sys
+
+from .signature import SourmashSignature
+from .logging import notify, error, set_quiet
+from .command_compute import (_compute_individual, _compute_merged,
+                              ComputeParameters)
+
+
+DEFAULTS = dict(
+    dna='k=31,scaled=1000,noabund',
+    protein='k=21,scaled=200,noabund',
```

Comment:
```suggestion
    protein='k=7,scaled=200,noabund',
```

Reply:
ok, so that's a question - what ARE the right defaults? :)

Reply:
(as recommended by our resident protein-space experts, Dr. Tessa Pierce and Olga Botvinnik :)

Reply:
Olga and I are finding that our best/preferred ksizes differ, which makes sense since our questions have different scope.

For protein, as `k=7` is a common protein k-mer size for existing applications, I'd vote for that as a default.
For dayhoff, I'm liking `k=19`, but Olga is using smaller. Perhaps `k=13` or `k=15` would be decent defaults?

Thoughts Dr. @olgabot?

Reply:
ok - so can I nominate k=10 for protein, k=16 for dayhoff? :) Any thoughts for hp?

Reply:
ok, seems our protein sizes have gotten closer! These defaults sounds good to me.